uk.co.compucorp.civicrm.hrcore/CRM/HRCore/Upgrader/Steps/1026.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+trait CRM_HRCore_Upgrader_Steps_1026 {
+
+  /**
+   * Deletes Default Groups
+   *
+   * @return bool
+   */
+  public function upgrade_1026() {
+    $this->up1026_deleteDefaultGroups([
+      'Advisory board',
+      'Newsletter Subscribers',
+      'Summer Program Volunteers',
+      'Administrators',
+    ]);
+
+    $this->up1026_disableAndHideDefaultGroup('Case_Resources');
+
+    return TRUE;
+  }
+
+  /**
+   * Deletes the Groups passed by params
```

Comment:
`@params` definition is missing from the comment section. Kindly provide it. 
I think `params` in the comment should be `parameter`.